File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    answer = [0]*2
    count_even = 0
    count_odd = 0
    
    for a in num_list:
        if a%2 ==0 :
            count_even +=1
        else:
            count_odd +=1
   
    answer[0]=count_even
    answer[1]=count_odd

    return answer

File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
   public int[] solution(int[] num_list) {
        int countEven = 0;
        for (int i : num_list) {
           if(i%2==0){
               countEven++;
           }
        }
        int[] answer = {countEven,num_list.length-countEven};
        return  answer;
    }
}